cmd/ci-reporter/cmd/github.go

```diff
@@ -76,18 +76,18 @@ func (r GithubReporter) GetCIReporterHead() CIReporterInfo {
 // CollectReportData implementation from CIReporter
 func (r GithubReporter) CollectReportData(cfg *Config) ([]*CIReportRecord, error) {
 	// set filter configuration
-	fieldFilter := map[FilteredFieldName][]FilteredBlacklistVal{}
+	blackListFilter := map[FilteredFieldName][]FilteredListVal{}
+	whiteListFilter := map[FilteredFieldName][]FilteredListVal{
```

Comment:
On Kubernetes, we see terminology such as white and black listing to be exclusionary and  problematic; instead we choose to use terms such as allowList and denyList. Better again, let's be even more semantically explicit about naming what issues *are* to be included and excluded from the report: so a `denyListFilter` could be called the `resolvedIssuesFilter`
and an `allowListFilter` could be called the `currentIssuesFilter` or the `openIssuesFilters` making post instantiation references clearer.

Reply:
Yes! Thanks for pointing this out